pdns/dnsdist-lua.cc

```diff
@@ -1477,6 +1477,20 @@ void setupLuaConfig(bool client)
       g_pools.setState(localPools);
     });
 
+  g_lua.writeFunction("setPoolBackupPolicy", [](string backupPool, string pool) {
```

Comment:
I think this Lua function is actually 'addPoolBackupPool'? and not 'set'? It also does not add a policy but a pool I think?

Reply:
Done! Renamed :)